README.md

```diff
@@ -4,12 +4,12 @@
 
 **- Batch Converter**: Batch convert CSV to Point Shapefile and other GIS formats (including OSM pbf, GeoPackage) in a folder and its subfolders;
 
-**- Download OpenData**:  Download Open Data from OSM Geofabrik, GADM, Microsoft Building Footprints, HCMGIS OpenData, Open Development Mekong, World Food Programme, Standford University, ect.
+**- Download OpenData**:  Download Open Data from OSM Geofabrik, GADM, Who's On First (WOF), Microsoft Building Footprints, HCMGIS OpenData, Open Development Mekong, World Food Programme, Standford University, ect.
 
 **- Vn-2000 Projections**:  Look up EPSG Code for Vn-2000/ TM3 Projections for all Provinces in Vietnam
 
 
-**- Spatial Processing**: 
+**- Spatial Processing**:
```

Comment:
Here and in many other places my IDE removed trailing whitespace. No-op in terms of functionality, but makes the DIFF far more complicated. I can restore the whitespace if desired, just let me know!